composer/datasets/in_context_learning_evaluation.py

```diff
@@ -138,6 +152,7 @@ def __init__(
         destination_path: str,
         question_prelimiter: str,
         fewshot_random_seed: int,
+        cot_delimiter: str = '',
```

Comment:
How do you differentiate between if a particular CoT setup is using their delimiter as `''` vs no CoT is being done? Is the first case even possible?

Reply:
Yes it is possible but there's no need to differentiate since the non-emptiness of the chain_of_thought key defines whether chain of thought is in effect